shared/AppInsightsCore/Tests/Unit/src/HelperFunc.Tests.ts

```diff
@@ -134,7 +134,7 @@ export class HelperFuncTests extends AITestClass {
             test: () => {
                 let newObject = objExtend();
                 Assert.ok(isObject(newObject), "The returned object is an object");
-                Assert.equal(0, objKeys(newObject), "The object should have no values");
+                Assert.equal(0, objKeys(newObject).length, "The object should have no values");
```

Comment:
This was part of the issue as `objKeys` was actually returning the function `Object.keys` and not the result.